osu.Game/Screens/SelectV2/PanelBeatmapStandalone.cs

```diff
@@ -222,7 +224,7 @@ protected override void PrepareForUse()
 
             var beatmapSet = beatmap.BeatmapSet!;
 
-            beatmapBackground.Beatmap = beatmaps.GetWorkingBeatmap(beatmap);
+            scheduledBackgroundRetrieval = Scheduler.AddDelayed(b => beatmapBackground.Beatmap = beatmaps.GetWorkingBeatmap(b), beatmap, 50);
```

Comment:
I remember testing this during original implementation but I couldn't see the overhead. I may have been incorrectly testing though, as working beatmaps do get cached after first load.

(And I guess that's one thing to note here – this is delayed the load even when they are cached already. Shouldn't change anything since you've just moved the delay from elsewhere, though.)